sormas-api/src/main/java/de/symeda/sormas/api/caze/CaseFacade.java

```diff
@@ -220,6 +222,10 @@ void saveBulkEditWithFacilities(
 
 	List<CaseDataDto> getByExternalId(String externalId);
 
+    default List<CaseDataDto> getCasesForWhichVaccinationIsRelevant(VaccinationDto vaccination){
```

Comment:
Incorrect code formatting; please ensure that you have the correct formatting settings and have enabled the save actions plugin (and set it to reformat your code on save)